docs/e2e/running.md

```diff
@@ -0,0 +1,59 @@
+#### Running e2e tests
+
+To run the e2e tests locally it is recommended to use the [e2e](https://quay.io/repository/openshift-on-azure/e2e-tests)
```

Comment:
This approach cannot possibly work when working locally with e2e tests, eg. you make a change in the test that you want to test out. I would suggest explaining how to run tests locally using ginkgo and having the docker image as an alternative option that does not require the user to use ginkgo or work in the context of openshift-azure in order to run the tests.

Reply:
fair enough